app/v2/controllers/AmendDisclosuresController.scala

```diff
@@ -56,7 +52,7 @@ class AmendDisclosuresController @Inject() (val authService: EnrolmentsAuthServi
         RequestHandler
           .withValidator(validator)
           .withService(service.amendDisclosures)
-          .withHateoasResult(hateoasFactory)(AmendDisclosuresHateoasData(nino, taxYear))
+          .withNoContentResult(OK)
```

Comment:
Instead of returning OK (200) we should be returning NO_CONTENT (204) when there is a no content result